src/librustc_mir/interpret/eval_context.rs

```diff
@@ -274,6 +280,20 @@ impl<'a, 'mir, 'tcx: 'mir, M: Machine<'a, 'mir, 'tcx>> EvalContext<'a, 'mir, 'tc
         }
     }
 
+    pub fn monomorphize_in_frame<T: TypeFoldable<'tcx> + Subst<'tcx>>(
```

Comment:
This name is odd, we don't have anything else called like it.  `monomorphize` would match better the fact that `resolve` and `subst_and_normalize_erasing_regions` also just implicitly depend on the frame.

Reply:
I swapped the naming scheme, so `monomorphize` is the one with only a type argument and `monomorphize_with_substs` doesn't need a stack frame to get the substs from.